### PR TITLE
03-1048: Show today's date and location on the clinic metrics header

### PR DESCRIPTION
**Pull request by makombe**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
Selection_026

*None.*

## Related Issue

https://issues.openmrs.org/browse/O3-1048

## Other

*None.*

### CODE REVIEW

File: packages/esm-outpatient-app/src/patient-queue-header/patient-queue-header.component.tsx [hunk 1]
```diff
@@ -15,8 +34,40 @@ const PatientQueueHeader: React.FC = () => {
...
     </div>
   );
 };
 
 export default PatientQueueHeader;
+
+function TodayDate() {
+  const today = dayjs(new Date());
+  return <span className={styles.dateLabel}>{today.format('DD  MMMM  YYYY')}</span>;
+}
```
Comment (donaldkibet) [inline packages/esm-outpatient-app/src/patient-queue-header/patient-queue-header.component.tsx:64]:
```suggestion

```
should use the helper date function pointed above

File: packages/esm-outpatient-app/src/patient-queue-header/patient-queue-header.component.tsx [hunk 2]
```diff
@@ -15,8 +34,40 @@ const PatientQueueHeader: React.FC = () => {
...
+
+function SessionLoction() {
+  const userSession = useSessionUser();
+  return userSession ? (
+    <span className={styles.locationLabel}>{userSession.sessionLocation.display}</span>
+  ) : (
+    <span></span>
+  );
+}
```
Comment (donaldkibet) [inline packages/esm-outpatient-app/src/patient-queue-header/patient-queue-header.component.tsx:73]:
```suggestion
const userSession = useUserSession()
userSession?.sessionLocation?.display
```
you can use optional chaining here

**Review by donaldkibet — CHANGES_REQUESTED** [review]
Thanks @makombe LGTM few adjustments needed

File: packages/esm-outpatient-app/src/patient-queue-header/patient-queue-header.component.tsx (hunk 2 above)
Comment (brandones) [reply packages/esm-outpatient-app/src/patient-queue-header/patient-queue-header.component.tsx:73]: Which also means, as I think @donaldkibet is implying, that you can get rid of this component and just inline it.

If you decide to keep it, fix the typo "SessionLoction" --> "SessionLoc**a**tion"

Comment (makombe) [reply packages/esm-outpatient-app/src/patient-queue-header/patient-queue-header.component.tsx:73]: @brandones and @donaldkibet  thanks for your reviews, I will act on them appropriately

**Pull request merged**